README/ReleaseNotes/v624/index.md

```diff
@@ -79,6 +79,21 @@ See the discussion at [ROOT-11014](https://sft.its.cern.ch/jira/browse/ROOT-1101
 - Just-in-time compilation of string expressions passed to `Filter` and `Define` now generates functions that take fundamental types by const value (rather than by non-const reference as before). This will break code that was assigning to column values in string expressions: this is an intended side effect as we want to prevent non-expert users from performing assignments (=) rather than comparisons (==). Expert users can resort to compiled callables if they absolutely have to assign to column values (not recommended). See [ROOT-11009](https://sft.its.cern.ch/jira/browse/ROOT-11009) for further discussion.
 - RDataFrame action results are now automatically mergeable thanks to the new interface provided by `ROOT::Detail::RDF::RMergeableValue` and derived, introduced in [#5552](https://github.com/root-project/root/pull/5552). A feature originally requested with [ROOT-9869](https://sft.its.cern.ch/jira/browse/ROOT-9869), it helps streamline RDataFrame workflow in a distributed environment. Currently only a subset of RDataFrame actions have their corresponding mergeable class, but in the future it will be possible to extend it to any action through the creation of a new `RMergeableValue` derived class.
 
+### Distributed computing with RDataFrame
+ROOT 6.24 introduces `ROOT.RDF.Experimental.Distributed`, an experimental python package that enhances RDataFrame with distributed computing capabilities. Users will be able to distribute their RDataFrame applications with the supported backend of their choice, while maintaining almost exactly the same code. The backend submodules of this package will expose their own `RDataFrame` objects. The only needed change in user code is to substitute `ROOT.RDataFrame` calls with such backend-specific `RDataFrame`s. Currently the [Apache Spark](http://spark.apache.org/) backend is supported, an example usage of this backend would look like:
+
+```python
+import ROOT
+
+# Point RDataFrame calls to the Spark specific RDataFrame
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
```

Comment:
Why not 

```suggestion
import ROOT.RDF.Experimental.Distributed.Spark.RDataFrame as RDF
```
?

Reply:
Long story. Short answer, we can't do it right now. Slightly longer answer, it's a bit of a mix between python packaging shenanigans and the way we expose the `ROOT` python module and the fact that it's "submodules" are actually cppyy namespaces.

It is possible (and I will propose a PR) to implement what you suggest, but would take some rework into PyROOT so it is left out of this first implementation for now

Reply:
As a first hint at the problem, see this:
```python
vpadulan@fedorathinkpad-T550 [~]: python
Python 3.8.7 (default, Jan 20 2021, 00:00:00) 
[GCC 10.2.1 20201125 (Red Hat 10.2.1-9)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ROOT.RDF
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'ROOT.RDF'; 'ROOT' is not a package
>>> import ROOT
>>> import ROOT.RDF
>>> 
```